src/components/half-earth-modal/half-earth-modal-component.jsx

```diff
@@ -43,7 +41,7 @@ const HalfEarthModalComponent = ({ handleModalClose, textData }) => {
         </div>
         <div className={styles.globeWrapper}>
           <div className={styles.progresBars}>
-            <img src={Globe} className={styles.globe} />
+            <img src={Globe} className={styles.globe} alt="" />
```

Comment:
Shouldn't we add some content to the `alt` attribute?

Reply:
@weberjavi Do you mean to put a string like "Half-Earth globe"?

Reply:
Yes @kfilip94 something like that, meaningful in case the image is not displayed or to the screen readers.

Reply:
@weberjavi  Done :)

Reply:
🙇